Jenkinsfile

```diff
@@ -12,7 +12,7 @@ pipeline {
           steps {
             container('jetty-build') {
               timeout( time: 120, unit: 'MINUTES' ) {
```

Comment:
You missed the timeout in the merge.

Reply:
no because I'm interested to have time log into the pod and get the logs :)

Reply:
oh you mean longer timeout?

Reply:
arfff :)

Reply:
arf got it